### ISSUE TITLE
Decoding tokens error - encountered fieldNum: 1, but we have already seen fnum: 5

### ISSUE THREAD

**Issue by petr-hejda** [binance-chain/javascript-sdk#168]
I'm getting following error while decoding list of tokens:

```
Error: encountered fieldNum: 1, but we have already seen fnum: 5
    at ./node_modules/@binance-chain/javascript-sdk/lib/decoder/index.js:139:15
    at Array.forEach (<anonymous>)
    at decodeObjectBinary (./node_modules/@binance-chain/javascript-sdk/lib/decoder/index.js:120:8)
    at decodeBinary (./node_modules/@binance-chain/javascript-sdk/lib/decoder/index.js:95:12)
    at decodeArrayBinary (./node_modules/@binance-chain/javascript-sdk/lib/decoder/index.js:191:26)
    at unMarshalBinaryBare (./node_modules/@binance-chain/javascript-sdk/lib/decoder/index.js:65:12)
    at Object.unMarshalBinaryLengthPrefixed (./node_modules/@binance-chain/javascript-sdk/lib/decoder/index.js:47:10)
    at Client._callee2$ (./node_modules/@binance-chain/javascript-sdk/lib/rpc/client.js:169:50)
    at tryCatch (./node_modules/regenerator-runtime/runtime.js:45:40)
    at Generator.invoke [as _invoke] (./node_modules/regenerator-runtime/runtime.js:271:22)
```

**Code to reproduce:**
```
const BnbApiClient = require('@binance-chain/javascript-sdk');

const RPC_URL = 'https://data-seed-pre-0-s3.binance.org/';
const RPC_NETWORK = 'mainnet';

const rpcClient = new BnbApiClient.rpc(RPC_URL, RPC_NETWORK);


rpcClient.listAllTokens(0, 4)
    .then(tokens => {
        console.log(tokens)
    })
    .catch(err => {
        console.error(err)
    })
```

---

It only happens if I try to list the 4th token.

**NOT OK:**
- `rpcClient.listAllTokens(0, 4)`
- `rpcClient.listAllTokens(0, 15)`
- `rpcClient.listAllTokens(1, 3)`
- `rpcClient.listAllTokens(1, 14)`

**OK:**
- `rpcClient.listAllTokens(0, 3)`
- `rpcClient.listAllTokens(1, 2)`
- `rpcClient.listAllTokens(4, 1)`

---

Version `"@binance-chain/javascript-sdk": "2.14.0"`

**Comment by rastislavkucera** [comment]
Still problem, version `2.14.6`.

**Comment by notatestuser** [comment]
There has been some discussion about this in #138.

Would you mind trying with [this fork](https://github.com/antoncoding/javascript-sdk/blob/master/src/decoder/index.js)?

**Comment by rastislavkucera** [comment]
> There has been some discussion about this in #138.
> 
> Would you mind trying with [this fork](https://github.com/antoncoding/javascript-sdk/blob/master/src/decoder/index.js)?

Tried `getTradingPairs(0,100)` , `listAllTokens(0,100)`, they return array with just 1 element, same with parameters `(1,1)`, `(0,1)`, `(0,10)`, `(1,2)`. Output is changing depending on `offset`

**Comment by xzjcool** [comment]
https://github.com/binance-chain/javascript-sdk/pull/198
Fixed in this pr

**Comment by xzjcool** [comment]
fixed in v2.14.7
